tests/functional/spec/dob_date.spec.js

```diff
@@ -0,0 +1,25 @@
+import DateOfBirthPage from "../generated_pages/dob_date/date-of-birth.page";
+import UnderSixteenPage from "../generated_pages/dob_date/under-sixteen.page";
+
+describe("Date of birth check", () => {
+  beforeEach("Load the survey", () => {
+    browser.openQuestionnaire("test_dob_date.json");
+  });
+  it("Given when I start survey, Then I see date of birth page", () => {
+    expect($(DateOfBirthPage.legend()).getText()).to.contain("date of birth");
+  });
+  it("Given Iam on date of birth page, when I enter date of birth less than 16 years, then on submit I get a message of under 16 in subsequent question title", () => {
```

Comment:
Probably needs a tweak in the text. I think we should try to avoid date of birth, as it's not a type. Affectively this is just routing on a date.

"Given I am completing a date question, when I enter a value less than 16 years ago, I am routed to the under 16 page"